contrib/win32/win32compat/win32_pty.c

```diff
@@ -90,10 +90,39 @@ CreateConPty(const wchar_t *cmdline,
 	return 0;
 }
 
-int is_conpty_supported()
+int
+is_conpty_supported()
 {
-	/* TODO - put conditional logic in here that determines if conpty is supported */
-	return 0;
+	wchar_t system32_path[PATH_MAX] = { 0, };
```

Comment:
Can you turn this into a singleton? We could just figure it out once whether ConPTY is supported/not.

Reply:
fixed